src/Razor/test/Microsoft.AspNetCore.Razor.Test.Common/Language/IntegrationTests/IntegrationTestBase.cs

```diff
@@ -491,12 +491,6 @@ protected void AssertSourceMappingsMatchBaseline(RazorCodeDocument codeDocument)
             {
                 var span = spans[i];
                 var sourceSpan = span.GetSourceSpan(codeDocument.Source);
-                if (sourceSpan == null)
-                {
-                    // Not in the main file, skip.
-                    continue;
-                }
-
```

Comment:
Huh?

Reply:
Nullability errors, it conplains that sourceSpan is not nullable so this condition is not needed.